README.md

```diff
@@ -19,46 +20,42 @@ More about it [here](https://www.terraform.io/language/expressions/type-constrai
 
 ```hcl
 module "my_pets_website_repo" {
-  source = "git::https://github.com/flaconi/terraform-github-repository.git?ref=master"
-
-  name = "my-pets-website"
+  source = "github.com/flaconi/terraform-github-repository.git?ref=master"
```

Comment:
Maybe we should advertise using released versions but I think it's pretty obvious...